ietf/name/fixtures/names.json

```diff
@@ -385,7 +385,7 @@
   },
```

Comment:
I've made the changes from this file to the production database. It would be better if this PR either didn't change this file at all, or update it to match production by fetching an updated database (after tonight's dump) and then running `ietf/manage.py generate_name_fixture --stdout | jq --sort-keys 'sort_by(.model, .pk)' >
ietf/name/fixtures/names.json`

Reply:
meh - nevermind - I'll take the action to update main after this is merged - I suspect other names need to be updated from other PRs anyhow.